e2e_tests/tests/cluster/test_users_experiment_api.py

```diff
@@ -1,53 +1,25 @@
-import pathlib
-
 import pytest
 
-import determined.common.api as determined_api
-import determined.common.api.certs as certs
-from determined.common import context, util
-from determined.common.api.bindings import determinedexperimentv1State as EXP_STATE
+from determined.common.api import bindings
+from determined.experimental import client
 from tests import config as conf
 from tests import experiment as exp
-
-from .test_users import ADMIN_CREDENTIALS, create_test_user, det_spawn, log_in_user
+from tests.cluster import test_users
 
 
 @pytest.mark.e2e_cpu
-def test_experimental_experiment_api_determined_disabled() -> None:
-    log_in_user(ADMIN_CREDENTIALS)
-    context_path = pathlib.Path(conf.fixtures_path("no_op"))
-    model_def_path = pathlib.Path(conf.fixtures_path("no_op/single-medium-train-step.yaml"))
-
-    model_context = context.read_legacy_context(context_path)
-
-    with model_def_path.open("r") as fin:
-        dai_experiment_config = util.safe_load_yaml_with_exceptions(fin)
+def test_experiment_api_determined_disabled() -> None:
+    test_users.log_in_user(test_users.ADMIN_CREDENTIALS)
 
     determined_master = conf.make_master_url()
-    user_creds = create_test_user(add_password=True)
+    user_creds = test_users.create_test_user(add_password=True)
 
     try:
-        det_spawn(["user", "deactivate", "determined"])
-        certs.cli_cert = certs.default_load(
-            master_url=determined_master,
+        d = client.Determined(determined_master, user_creds.username, user_creds.password)
```

Comment:
I think this test is supposed to be disabling the default `determined` user, and it got lost in refactoring

Reply:
excellent catch, will fix